protodata/src/main/proto/spine/validation/rule.proto

```diff
@@ -0,0 +1,138 @@
+syntax = "proto3";
+
+package spine.validation;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.spine.io";
+option java_package = "io.spine.validation";
+option java_outer_classname = "RuleProto";
+option java_multiple_files = true;
+
+import "spine/validation/value.proto";
+import "spine/protodata/ast.proto";
+import "google/protobuf/any.proto";
+
+// A simple validation rule.
+//
+// May be expressed as a comparison between two operands. The first operand is always a field value.
+// The second operand is the `other_value`. `other_value` is typically derived from the Protobuf
+// definition of the validation rule.
+//
+message Rule {
+
+    // The field which is being compared.
+    protodata.Field field = 1;
+
+    oneof operation {
+
+        // A simple comparison sign.
+        Sign sign = 2;
```

Comment:
To me, this is an `ComparisonOperator`. `Sign` is too wide to me, as "-7" is also a sign and a number, while it's clear it's far from comparison.